### ISSUE TITLE
Site Editor: Apply mouse click changes to editor mode to the entire area

### ISSUE THREAD

**Issue by t-hamano** [WordPress/gutenberg#48096]
## What problem does this address?

In the site editor, you can enter the edit mode by clicking on the content area. However, this click is limited to only where the block exists. Therefore, if the template has little content, I think it would confuse the user, since most areas will not happen when clicked. If there is no block in the template, the click will not work at all and the user will have to press the edit button.

## What is your proposed solution?

I believe that edit mode triggers should be applied to all areas of an area, regardless of the amount of content.

## Screencast

https://user-images.githubusercontent.com/54422211/219018566-91058aba-4a65-43d3-bbb2-d7b3f0b0ff20.mp4

**Comment by t-hamano** [comment]
I have tested this issue again with the latest trunk. As another example, if the content is empty, it will be more difficult to enter the editor mode with a mouse click.

https://github.com/WordPress/gutenberg/assets/54422211/343577d8-75f3-4271-9c3f-bb3c5b75510d

**Comment by t-hamano** [comment]
I retested this issue and found that it no longer occurs.

- The entire editor area is clickable, even for empty templates or templates with little content.
- For template parts and patterns, with the introduction of the DataViews, there is no longer a detail page. The entire preview area in the DataViews is clickable.

https://github.com/user-attachments/assets/1096931f-5197-4961-a279-5ff3f9d4fd9e
